python/cudf/tests/test_hdf.py

```diff
@@ -0,0 +1,150 @@
+# Copyright (c) 2018, NVIDIA CORPORATION.
+
+import cudf
+from cudf.tests.utils import assert_eq
+
+import pytest
+import os
+import pandas as pd
+import numpy as np
+from string import ascii_letters
+
+
+try:
+    import tables # noqa F401
+    _have_pytables = True
+except ImportError:
+    _have_pytables = False
```

Comment:
Same question about the flag and pytest.skip

Reply:
same as above